frontend/src/modules/app/ToolBar.tsx

```diff
@@ -363,12 +365,8 @@ export const ToolBar = ({
             </Grid>
             <Grid item className={classes.marginContent}>
               {alarmConfig.map((param: AlarmConfiguration) => {
-                if (alarmLimits !== null && alarmLimitsStandby !== null) {
-                  if (
-                    alarmLimits[param.stateKey].lower !==
-                      alarmLimitsStandby[param.stateKey]?.lower ||
-                    alarmLimits[param.stateKey].upper !== alarmLimitsStandby[param.stateKey]?.upper
-                  ) {
+                if (alarmLimits !== null && alarmLimitsDraft !== null) {
```

Comment:
Is this line actually needed? In AlarmsPage.tsx it seems to have been omitted.

Reply:
Yes! Toolbar is actually on landing page too, and I think it would be null initially and frontend crashes saying these are null which is not the case in AlarmsPage

Reply:
Ah, ok!